db/migrate/20231210065725_create_languages.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLanguages < ActiveRecord::Migration[7.0]
```

Comment:
what are these going to be used for? is it regular languages or programming ones?